src/lib.rs

```diff
@@ -6,43 +6,53 @@ use std::{fs, process};
 
 use anyhow::{bail, ensure, Context, Result};
 use notify::{RecommendedWatcher, RecursiveMode, Watcher};
-use structopt::StructOpt;
+use structopt::{lazy_static, StructOpt};
 use wasm_bindgen_cli_support::Bindgen;
 
+pub fn get_metadata() -> &'static cargo_metadata::Metadata {
```

Comment:
`get` is redundant. It's always getting something
```suggestion
pub fn metadata() -> &'static cargo_metadata::Metadata {
```